gui/src/app/pages/HomePage/LoadProjectWindow.tsx

```diff
@@ -142,6 +174,45 @@ const LoadProjectWindow: FunctionComponent<LoadProjectWindowProps> = ({
           </Button>
         </div>
       )}
+      <h3>Load from browser</h3>
+      {browserProjectTitles.length > 0 ? (
+        <table>
+          <tbody>
+            {browserProjectTitles.map((title) => (
+              <tr key={title}>
+                <td>
+                  <SmallIconButton
+                    icon={<Delete />}
+                    onClick={async () => {
+                      const ok = window.confirm(
+                        `Delete project "${title}" from browser?`,
+                      );
+                      if (!ok) return;
+                      const bpi = new BrowserProjectsInterface();
+                      await bpi.deleteProject(title);
+                      const titles = await bpi.listProjects();
+                      setBrowserProjectTitles(titles);
+                    }}
+                  />
+                </td>
+                <td>
+                  <Link
+                    onClick={() => {
+                      handleOpenBrowserProject(title);
+                    }}
+                    component="button"
+                    underline="none"
+                  >
+                    {title}
```

Comment:
I agree it would be helpful. But I'm wary about including it in meta.json as there's nothing that is enforcing the accuracy of that information if someone edits the project outside of SP. And for Gists, this information would be available independent of the content of meta.json.

Let me put the timestamp in the browser storage record and we'll see how that looks.

Reply:
I think this is a good idea (modulo all the usual complications about timestamps/time zones) but might be for a different PR? I'd want to see the other forms of persistence be aware of it also.

Reply:
(Didn't see your message Jeff until just now.)

I added timestamps for the saved browser projects, but it's not in meta.json.

The timestamps are now shown (in the form of "time-ago-strings", like "3 min ago").

Reply:
![image](https://github.com/user-attachments/assets/0da162eb-923a-444b-911c-e12e9f7b8c35)